integreat_cms/cms/migrations/0038_add_organization_permissions_to_management_role.py

```diff
@@ -0,0 +1,70 @@
+# Generated by Django 3.2.11 on 2022-05-18 00:05
+from django.db import migrations
+
+
+NEW_MANAGEMENT_PERMISSIONS = [
+    "change_organization",
+    "view_organization",
+    "delete_organization",
+]
+# pylint: disable=unused-argument
+def add_organization_mangement_permissions(apps, schema_editor):
+    """
+    Add organization permissions to management role
+
+    :param apps: The configuration of installed applications
+    :type apps: ~django.apps.registry.Apps
+
+    :param schema_editor: The database abstraction layer that creates actual SQL code
+    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
+    """
+    # We can't import the Person model directly as it may be a newer
+    # version than this migration expects. We use the historical version.
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+
+    management_group = Group.objects.filter(name="MANAGEMENT").first()
+    management_group.permissions.add(
+        *Permission.objects.filter(codename__in=NEW_MANAGEMENT_PERMISSIONS)
+    )
+
+
+# pylint: disable=unused-argument
+def remove_organization_mangement_permissions(apps, schema_editor):
+    """
+
+    Remove organization permissions from management role
+
+    :param apps: The configuration of installed applications
+    :type apps: ~django.apps.registry.Apps
+
+    :param schema_editor: The database abstraction layer that creates actual SQL code
+    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
+    """
+    # We can't import the Person model directly as it may be a newer
+    # version than this migration expects. We use the historical version.
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+
+    management_group = Group.objects.filter(name="MANAGEMENT").first()
+    management_group.permissions.remove(
+        *Permission.objects.filter(codename__in=NEW_MANAGEMENT_PERMISSIONS)
+    )
```

Comment:
```suggestion
    Group = apps.get_model("auth", "Group")
    Permission = apps.get_model("auth", "Permission")

    # Clear and update permissions according to new constants
    for role_name in dict(roles.CHOICES):
        group = Group.objects.filter(name=role_name).first()
        # Clear permissions
        group.permissions.clear()
        # Set permissions
        group.permissions.add(
            *Permission.objects.filter(codename__in=roles.PERMISSIONS[role_name])
        )
```
Since we're changing permissions quite frequently, I guess it's justifiable to use a more generic approach here (with the advantage that we can always be 100% sure that they are in sync with the constants module and the disadvantage that we cannot reverse them)...

Reply:
Yes, but I think it is kind of odd since it is basically the same as migration no. 25. But if you want we can do it that way

Reply:
Yeah, merging those two won't work for the [same reason](https://github.com/digitalfabrik/integreat-cms/pull/1712#discussion_r979807938) we cannot put the code in the initial role migration `0002_roles.py`: The production system already knows that it has applied the migration `0025_update_roles.py` and won't apply it again, even if we change it.
In the end, the explicit role change you suggested is better 1. because it's more clear what happens and 2. it's reversible, but it's also more work, so if you don't want to add the role for the service team manually as well, I'm also fine with the more generic approach (which just needs to be copied + pasted each time a permission is changed)...

Reply:
Makes sense, so I did implement it in the lazy way now :D